source/core/CEntryPoints.cpp

```diff
@@ -526,11 +526,16 @@ VIREO_EXPORT Boolean TypeRef_IsAnalogWaveform(TypeRef typeRef)
     return typeRef->IsAnalogWaveform();
 }
 //------------------------------------------------------------
-VIREO_EXPORT Boolean TypeRef_IsJavascriptRefnum(TypeRef typeRef)
+VIREO_EXPORT Boolean TypeRef_IsJavaScriptRefNum(TypeRef typeRef)
```

Comment:
Can we make a change so that we have both TypeRef_IsJavaScriptDynamicRefnum and TypeRef_IsJavaScriptStaticRefNum?

It maps better to how the types are actually implemented in Vireo as JavaScriptStaticRefnum and JavaScriptDynamicRefnum are distinct different types (there is not a JavaScriptRefnum type).

We could also have a helper TypeRef_IsJavaScriptRefNum if it's valuable but there are so few usages I don't think it is necessary. 

The following call site in the typeVisitor would have to be updated to check both cases: https://github.com/ni/VireoSDK/blob/master/source/core/module_typeHelpers.js#L260

Talking to Carlos there may be benefit in the future of making the typeVisitor being aware of static vs dynamic types as well. One example is that the JSLI's type visitor (that is user facing) could enforce that only dynamic references are allowed while the Editor side type visitor (for internal functions) would take both static and dynamic references. We won't make that change now as that would be a public api change.

Reply:
Done! Thanks!